<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dicom_Viewer</class>
 <widget class="QDialog" name="Dicom_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>705</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dicom_Viewer</string>
  </property>
  <widget class="QPushButton" name="Browse_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Browse Image</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Axial_Plane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Sagittal_Plane">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Coronal_Plane">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Diagonal_Plane">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Axial_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Axial Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sagittal_Label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sagittal Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Coronal_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Coronal Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Diagonal_Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Oblique Plane:</string>
   </property>
  </widget>
  <widget class="QSlider" name="AxialHorizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSlider" name="CoronalHorizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="SagittalHorizontalSlider">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="DiagonalHorizontalSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="AxialVerticalSlider">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="CoronalVerticalSlider">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="SagittalVerticalSlider">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="DiagonalVerticalSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="obliqueLineSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="obliqueLineSlopeSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lineLength_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Length of the line = </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
